src/pages/Loans/Loans.js

```diff
@@ -20,25 +20,117 @@ import {
 	getContractType,
 	getLoansCollateralPair,
 } from '../../ducks/loans/contractInfo';
+import Actions, { ActionTypes } from './components/Actions';
+import ModifyCollateral from './components/LoanCards/ModifyCollateral';
+import LiquidateCard from './components/LiquidateCard';
+import { getNetworkId } from 'ducks/wallet/walletDetails';
 
-const Loans = ({ collateralPair, fetchLoansContractInfo, contractType }) => {
+import Notify from 'bnc-notify';
+
+export const VIEWS = {
+	LOANS: 'loan',
+	LIQUIDATIONS: 'liquidations',
+};
+
+const Loans = ({ collateralPair, fetchLoansContractInfo, contractType, networkId }) => {
 	const [selectedLoan, setSelectedLoan] = useState(null);
+	const [selectedLiquidation, setSelectedLiquidation] = useState(null);
+	const [visiblePanel, setVisiblePanel] = useState(null);
+	const [view, setView] = useState(VIEWS.LOANS);
+	const [notify, setNotify] = useState(null);
+
+	useEffect(() => {
+		var notify = Notify({
+			dappId: '95a4ea13-9af6-4ea1-89db-a2c333236a77',
```

Comment:
@evgenyboxer can you help me set up the process.env for the actual keys, I can pm you it.

Reply:
@clementbalestrat pls help replace these keys to the one you sent me